### DISCUSSION TITLE
Everspring - EH403 - Cannot change default configuration

### DISCUSSION THREAD

**Discussion by s8109csa** [zwave-js/zwave-js#6828] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I cannot change default configuration on Everspring EH403. I have 3 of them, same issue on all. There are 3 parameters and for me the important would be the auto report lux value. I wanted to change 0 to 3 and when I clicked somewhere else in the Home Assisstant, there was only an error message: Error happened. Other Z-Wave devices are working as expected.

What did you expect to happen instead?
Update parameters

  1. Go to Devices -> Z-Wave -> Everspring EH403 -> Settings -> Edit parameter 3 -> click out from the cell.
  2. See error

Anything else we should know?
The devices are securely connected to Aeotec USB Z-Stick 7 - again, I can edit other devices. FW is default, I don't think there was ever a firmware update on these flood lights.

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...newest but it was always the case

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: EverSpring
Model name: EH-403
Node ID: 11

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[home-assistant_zwave_js_2024-05-08T15-09-55.901Z.log](https://github.com/zwave-js/node-zwave-js/files/15250841/home-assistant_zwave_js_2024-05-08T15-09-55.901Z.log)

**Comment by AlCalzone** [comment]
Any chance you have a manual for that device you can share?

**Reply by s8109csa** [reply]
[EH403-Manual_A501112670R02-0423.pdf](https://github.com/zwave-js/node-zwave-js/files/15252892/EH403-Manual_A501112670R02-0423.pdf)
Absolutely and thank you for your support in advance.
Luckily my issue is not huge as I am able to query the device every x minutes with an automation, so no big pain, but would be nice if this would work perfectly.

Peter

**Comment by AlCalzone** [comment]
Ok the parameter that is sent matches the manual, so it's likely another device with broken Supervision implementation. We'll have to disable the use of that CC for this device.

**Reply by AlCalzone** [reply, marked as answer]
Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/6829
